app/assets/javascripts/districts.js

```diff
@@ -0,0 +1,244 @@
+const controlledPatientData = JSON.parse('<%= raw(@controlled_patients.to_json) %>');
```

Comment:
@rsanheim is this the right place to save the `js` for reporting views?

Reply:
Yup this should be right. You may have to include it in the layout via a JavaScript include tag, I don’t remember if Rails does that for you via convention or not.